near/Dockerfile.build

```diff
@@ -3,8 +3,10 @@ FROM debian@sha256:2ce44bbc00a79113c296d9d25524e15d423b23303fdbbe20190d2f96e0aeb
 RUN apt-get update && apt-get install apt-utils && apt-get install -y python3 npm curl --no-install-recommends
 RUN apt-get install -y build-essential git
 
-ADD . .
+ADD setup-rust.sh .
 RUN ./setup-rust.sh
+ADD . .
+RUN make clean
```

Comment:
Just out of curiosity, do you need to "make clean" on a brand spanking new docker?  Or does this cover rebuild/cache cases?

Reply:
The add . .   copied in all my prebuild files and I wanted them to be built entirely in the dockerfile

Reply:
you can explicitly ignore those in the `.dockerignore` file next to the docker file